app/controller.go

```diff
@@ -320,12 +318,10 @@ func (app *App) blockEnder() blockEnder {
 		ethTrackerlog := log.NewLoggerWithPrefix(app.Context.logWriter, "ethtracker").WithLevel(log.Level(app.Context.cfg.Node.LogLevel))
 		doTransitions(app.Context.jobStore, app.Context.btcTrackers.WithState(app.Context.deliver), app.Context.validators)
 		doEthTransitions(app.Context.jobStore, app.Context.ethTrackers, app.Context.node.ValidatorAddress(), ethTrackerlog, app.Context.witnesses, app.Context.deliver)
-		// Proposals currently in store are cleared if deliver is successful
-		// If Expire or Finalize TX returns false,they will added to the proposals queue in the next block
-		// Errors are logged at the function level
-		// These functions iterate the transactions store
-		ExpireProposals(&app.header, &app.Context, app.logger)
```

Comment:
Do u want to remove the previously used functions like `ExpireProposals` from `app/internalTx.go`?